### PR TITLE
djs: removed member ID and push call

### PR DESCRIPTION
**Pull request by dylanjshaw001**
- remove unused blackcrow constant
- remove data layer push containing member ID
- removed member ID field from configs

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -66,14 +59,6 @@ const copyFromWindow = require('copyFromWindow');
 const callInWindow = require('callInWindow');
 const createQueue = require('createQueue');
```
Comment (adriennepaquin) [inline template.tpl]: do we want to keep these definitions if they're no longer being used?

**Review by adriennepaquin — APPROVED** [review]
makes sense to me. Just one comment/question

File: template.tpl (hunk 1 above)
Comment (dylanjshaw001) [reply template.tpl]: duh, good call! just removed

**Review by adriennepaquin — APPROVED** [review]

File: template.tpl [hunk 2]
```diff
@@ -128,106 +110,6 @@ ___WEB_PERMISSIONS___
       "isEditedByUser": true
     },
     "isRequired": true
-  },
```
Comment (joelgoldman) [inline template.tpl:131]: What were these lines of code for?

File: template.tpl [hunk 3]
```diff
@@ -16,7 +16,7 @@ ___INFO___
   "displayName": "Black Crow AI",
   "brand": {
     "id": "github.com_october8ai",
-    "displayName": "Black Crow AI",
+    "displayName": "october8ai",
```
Comment (joelgoldman) [inline template.tpl]: Why did we change the displayName?

File: template.tpl (hunk 2 above)
Comment (dylanjshaw001) [reply template.tpl:131]: Removing the member ID field/functionality because it's no longer necessary and can cause issues for partners

**Review by joelgoldman — APPROVED** [review]

**Pull request merged**